web-common/src/Data/Lens/Extra.purs

```diff
@@ -14,3 +18,9 @@ import Data.Maybe.First (First)
 -- something in a relaxed way."
 peruse :: forall m s t a b. MonadState s m => Fold (First a) s t a b -> m (Maybe a)
 peruse = gets <<< preview
+
+-- | Analagous to `toListOf`. This is included in a forthcoming
+-- release of purescript-profunctor-lenses. When we update we can delete
+-- this.
+toArrayOf :: forall s t a b. Fold (Endo (->) (List a)) s t a b -> s -> Array a
+toArrayOf p = Array.fromFoldable <<< toListOf p
```

Comment:
👍

Reply:
that this is missing annoyed me recently

Reply:
Yeah. It's committed to the library but not yet release...